compose-production.yml

```diff
@@ -21,18 +21,21 @@ webapp:
   links:
     - rabbitmq:rabbit
     - database
-    - email
     - celery
   environment:
     - APP_PORT=8000
     - APP_HOST=0.0.0.0
     - DJANGO_CELERY_BROKER_URL=amqp://guest@rabbit
     - DJANGO_DATABASE_URL=postgis://postgres@database/postgres
-    - DJANGO_EMAIL_USE_TLS=false
-    - DJANGO_EMAIL_HOST=email
-    - DJANGO_EMAIL_HOST_PASSWORD=osmaxx
-    - DJANGO_EMAIL_HOST_USERNAME=osmaxx
-    - DJANGO_EMAIL_PORT=25
+    - DJANGO_EMAIL_BACKEND=django.core.mail.backends.smtp.EmailBackend
+    - DJANGO_EMAIL_HOST=<changeme>
+    - DJANGO_EMAIL_HOST_PASSWORD='<changeme>'
+    - DJANGO_EMAIL_HOST_USER='<changeme>'
+    - DJANGO_EMAIL_PORT=<changeme>
+    - DJANGO_EMAIL_SUBJECT_PREFIX='<changeme>'
+    - DJANGO_EMAIL_USE_TLS=<changeme>
+    - DJANGO_EMAIL_USE_SSL=<changeme>
+    - DJANGO_EMAIL_TIMEOUT=<changeme>
```

Comment:
You're right, it would run, even docker compose would start. Wrong thinking on my end, I'll change that to documentation and simple configuration.